epoxy-paging/src/main/java/com/airbnb/epoxy/paging/PagingEpoxyController.java

```diff
@@ -177,6 +177,24 @@ public void setList(@Nullable PagedList<T> list) {
     updatePagedListSnapshot();
   }
 
+  /**
+     * Returns the list currently being displayed by the EpoxyController.
```

Comment:
can you note that this is either a snapshot of the paged list or the list set with `setList`?

Reply:
Oh! you are right, this is inaccurate